easybuild/easyblocks/r/rosetta.py

```diff
@@ -234,6 +234,8 @@ def extract_and_copy(dirname_tmpl, optional=False):
         # (extract and) copy database and biotools (if it's there)
         extract_and_copy('rosetta_database%s')
         extract_and_copy('BioTools%s', optional=True)
+        extract_and_copy('rosetta_tools%s')
```

Comment:
are you sure this is always there? should we have `optional=True` included to just ignore it if it's not there?